server/src/controllers/authTeamController.js

```diff
@@ -0,0 +1,102 @@
+import rp from 'request-promise';
+
+const server = 'http://localhost:8080';
+
+// Triggered from 'GET /slack/teams/auth' after bot was added to team
+// Queries Slack and makes a request to teamcontroller to add team
+const authTeam = (req, res) => {
+  let options = {
+    uri: 'https://slack.com/api/oauth.access',
+    method:'GET',
+    qs: {
+      client_id: process.env.CLIENT_ID,
+      client_secret: process.env.CLIENT_SECRET,
+      code: req.query.code,
+      redirect_uri: `${server}/slack/teams/auth`
+    },
+    json: true
+  }
+
+  console.log('----- Making Request to Slack -----');
+  rp(options)
+  .then((body) => {
+    if (body.ok) {
+      let slackTeamData = {
+        slackTeamToken: body.access_token,
+        slackTeamName: body.team_name,
+        slackTeamId: body.team_id,
+        slackBotId: body.bot.bot_user_id,
+        slackBotToken: body.bot.bot_access_token
+      }
+      return slackTeamData;
+    } else {
+      res.redirect('/oops');
```

Comment:
do we have any specific oops handling set up, or is this just gonna be a general error page?